osquery/tables/system/linux/services.h

```diff
@@ -0,0 +1,169 @@
+/**
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under both the Apache 2.0 license (found in the
+ *  LICENSE file in the root directory of this source tree) and the GPLv2 (found
+ *  in the COPYING file in the root directory of this source tree).
+ *  You may select, at your option, one of the above-listed licenses.
+ */
+
+#pragma once
+
+#include <osquery/core.h>
+#include <osquery/logger.h>
+
+/*
+  systemd does not support static linking; not only it doesn't work, but they
```

Comment:
This scares me a lot. If the project is actively fighting static linking we should find out why.

Reply:
They do not support it, so they want to add a clear error as to why it doesn't work when people try to enable the static option in the project. We are using the dynamic library that is present on the system here, and everything seems to be working fine (I didn't test it with Ubuntu 18 though because it wasn't yet available)